### PR TITLE
DisableNotifications button added in NotificationTexDiv of NotificationBotPolyon

### PR DESCRIPTION
**Pull request by askaborg**
- Headsup display on top of each screen as you make your way through the app
What this will show:
- What you are working on > what is next > Track progress
- When the user logs in, this would pop-up and then disappear 

---
Add a summary that address the following:
 - motivation/purpose for the feature 
     To give expert users the ability to disable and not be bothered by notifications.
 - What were those changes
     A disable notifications button was added beside the close button of the NotificationBotPolyon.js. 
 - What issues were fixed
     Disable notifications allows the user to focus.
 - Provide the link to the user story it is referencing in Trello board or other supporting document
     https://trello.com/c/elTDaDii/21-as-a-user-i-want-to-change-notification-settings-so-that-i-can-mute-all-notifications

### CODE REVIEW

**Comment by Jtamedrano** [comment]
Something wasn't passing here. can we see what it is before pushing?

**Comment by tinomen** [comment]
Feel free to remove the template questions.

**Comment by tinomen** [comment]
In the future let snot commit empty files. It's not at all clear why it's there and the commit will have not value on your activity.

File: temp.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This file will log notes for commits and removed when the feature is complete.
```
Comment (tinomen) [inline temp.txt]: Lets not do this in the future. It looks like you are trying to game the github activity graph. But it's really adding noise to the team members who have to review this.

**Review by tinomen — CHANGES_REQUESTED** [review]
This PR is empty. None of the changes described are present.

**Comment by tinomen** [comment]
Closing because there is nothing here.

**Pull request closed without merging**